src/Apps/CollectorProfile/Routes/Saves2/CollectorProfileSaves2Route.tsx

```diff
@@ -40,6 +41,10 @@ const CollectorProfileSaves2Route: FC<CollectorProfileSaves2RouteProps> = ({
 
   return (
     <>
+      <SavesHeader />
+
+      <Spacer y={4} />
+
       <Shelf showProgress={false}>
```

Comment:
For some reason, `Shelf` component adds 60px margin below:

<img width="1035" alt="Screenshot 2023-02-16 at 15 36 19" src="https://user-images.githubusercontent.com/3934579/219395080-80eff914-79f0-43b5-97f6-f6439c96913e.png">

There is no such margin in the design. How do I get rid of it? I've tried passing a `style` property, but TS compiler was complaining that `style` is not a part of an interface. Do I have to add a `marginBottom` property in Palette or is there an easier way?

Reply:
cc @dzucconi

Reply:
This would be a bug with the `showProgress` prop (I didn't know this existed, are we sure we want to turn off the scrollbar? Seems like a very bad idea). I can fix but, TBH, I'd rather remove that prop?

Reply:
> Is it true that, as a consequence, everyone would see the solid scrollbar as [in these stories](https://palette-storybook.artsy.net/?path=/story/components-shelf--differing-amounts) until there was overflow?

That's right. Though if you don't want that then a simple: `<Box overflowX="auto" style={{ WebkitOverflowScrolling: "touch" }}>` is probably the way to go.

Reply:
Thanks all, will clarify and address in another PR